PyExpLabSys/common/database_saver.py

```diff
@@ -523,11 +530,12 @@ def run(self):
                     success = True
                     SQL_SAVER_LOG.debug('Executed query\n\'%.70s\'\nwith args: %.60s', query,
                                         args)
-                except MySQLdb.OperationalError: # Failed to perfom commit
-                    SQL_SAVER_LOG.error(
-                        'Executing a query raised an MySQLdb.OperationalError. Make new '
-                        'database connection and retry in 5 seconds.'
-                    )
+                # Naming exceptions is a bit tricky here, since we import different
+                # sql-libraries at runtime, not all exceptions are available and we
+                # end up with a NameError instead. Catching all should be ok here.
+                except Exception as e: # Failed to perfom commit
```

Comment:
I see the point. This seems a pragmatic approach if we are are continue supporting multiple DB backends.